docs/helpers.rst

```diff
@@ -261,65 +285,23 @@ Example
         assert list(m.to) == ['to@example.com']
 
 
-Environment autouse fixtures
-----------------------------
+Automatic cleanup
```

Comment:
I changed this title because they appear to the end developer not as fixtures but as general features of pytest-django.